src/actors/runtime/AbstractActor.ts

```diff
@@ -52,7 +57,7 @@ export default abstract class AbstractActor {
   constructor(daprClient: DaprClient, id: ActorId) {
     this.daprClient = daprClient;
     this.actorClient = new ActorClient(daprClient.getDaprHost(), daprClient.getDaprPort(), daprClient.getCommunicationProtocol(), daprClient.getOptions());
-
+    this.logger = new Logger(daprClient.getOptions().logger)
```

Comment:
I would include the component name and area in here

e.g. 

```javascript
private readonly logger = new Logger(MyService.name);
```

ref: https://docs.nestjs.com/techniques/logger#using-the-logger-for-application-logging

Reply:
Hmm, we won't be able to pass around Logger object anymore. We will need to pass around Logger options instead and have each class instantiate their own Logger based on `LoggerOptions`.

Reply:
This has been updated to include component/area during the initialization. As discussed, logger is no longer passed explicitly. HttpClient options exposes logger options that is reused.